cameo/flux_analysis/structural.py

```diff
@@ -191,6 +190,13 @@ def find_coupled_reactions_nullspace(model, ns=None, tol=1e-10):
             reaction_j = non_blocked_reactions[j]
             right = non_blocked_ns[j]
             ratio = np.apply_along_axis(lambda x: x[0] / x[1] if abs(x[1]) > 0. else np.inf, 0, np.array([left, right]))
+
```

Comment:
It cannot be done in the same step, because it only applies if the ratio between all other elements is 1 and it is only true if `left[i] == 0` and `right[i] == 0`.